### ISSUE TITLE
WCAG compliance

### ISSUE THREAD

**Issue by asivath** [asivath/package-rater#3]
WCAG will meet the requirements of the ADA (ADA guidance from the DoJ). Your website should be compliant with WCAG 2.1 at level AA. See https://www.w3.org/TR/WCAG21/ for details. These free tools from Microsoft may be helpful. (make it pretty now)
Test using setup for Selenium achieving 80% minimum score
